docs/content/reference/ransomware.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Ransomware"
+date: 2021-06-23T18:13:59+05:30
+draft: true
+pre: '<i class="fas fa-lock"></i> '
+weight: 10
+---
+
+The Infection Monkey has the capability of simulating a ransomware attack on your network.
+All actions performed by the encryption routine are designed to be safe for production
+environments.
+
+To ensure minimum interference and easy recoverability, the ransomware simulation will encrypt
+files only if the user specifies a directory that contains files that are safe to encrypt.
+If no directory is specified, no files will be encrypted.
+
+<!-- add config screenshot here -->
+
+
+## How are the files encrypted?
+
+Files are "encrypted" in place with a simple bit flip. Encrypted files are renamed to have
+`.m0nk3y` appended to their names.
+
+This is a safe way to simulate encryption since it is easy to "decrypt" your files. You can simply perform a bit flip on the files again and rename them to remove the appended `.m0nk3y` extension.
+
+This is sufficient to mock a ransomware attack on your network as your files are left unusuable and are renamed with a different extension, similar to the way that many ransomwares act. As this is a simulation, your security solutions should be triggered to notify and prevent these changes from taking place.
```

Comment:
```suggestion
This is sufficient to mock a ransomware attack on your network as your files are left "unusable" and are renamed with a different extension, similar to the way that many ransomwares act. As this is a simulation, your security solutions should be triggered to notify and prevent these changes from taking place.
```

Reply:
The users need to be sure that the files are not actually unusable.

Reply:
I think the paragraph right before this one specifies that well enough?

Reply:
I think the paragraph right before this one specifies that clearly enough. WDYT?

Reply:
What if we say "the data in the files has been manipulated" instead of "your files are left unusable".

Reply:
> I think the paragraph right before this one specifies that clearly enough. WDYT?

Yes it does. I just said so the user reading this will not be scared.

I like what @mssalvatore suggested.

Reply:
How about now?
https://github.com/guardicore/monkey/pull/1265/commits/954cc469cf7cab9c50c18b312fa9741895abd235

Reply:
I'm good with this change. I'm going to have marketing review this before it gets released as well, so I don't think we have to agonize over every word :)